### DISCUSSION TITLE
Scriptable Profiles

### DISCUSSION THREAD

**Discussion by bmpitzen** [jordanbaird/Ice#108] [General]
I saw that profiles were on your roadmap. A feature that would be super nice is somehow exposing them to be scriptable either with keyboard shortcuts or through shortcuts or something like that.

**Comment by jordanbaird** [comment]
Eventually, I'd like the entire app to be scriptable. Probably through AppleScript first, and then we can expose the key pieces of functionality to Shortcuts
